llumnix/backends/vllm/worker.py

```diff
@@ -145,3 +146,14 @@ def shutdown(self) -> None:
     def restart(self) -> None:
         self.init_model()
         self.init_cache_engine(self.cache_config)
+
+    # Note: this Method is only used for testing !!!
+    def set_gpu_cache(self, data):
```

Comment:
We should implements  these functions in a `TestClass`

Reply:
get
see https://github.com/AlibabaPAI/llumnix/pull/22